src/store/eventing/__tests__/fetch-test.ts

```diff
@@ -0,0 +1,15 @@
+import eventManager from '../../../common/EventManager';
+import { AppState } from '../../types';
+import { AsyncAction } from '../types';
+import { handleFetchErrorEvents } from '../fetch';
+
+jest.mock('../../../common/EventManager');
+
+describe('store/eventing/fetch', () => {
+    test('should emit fetch error event', () => {
+        const error = new Error('fetch');
+        handleFetchErrorEvents({} as AppState, {} as AppState, { error } as AsyncAction);
+
+        expect(eventManager.emit).toBeCalledWith('annotations_fetch_error', { error });
```

Comment:
Should this use the enum for the event action?

Reply:
> Should this use the enum for the event action?

I looked into the other tests and found they are all using plain strings. So I used plain string for consistency.